cmd/utils/executor.go

```diff
@@ -70,10 +71,11 @@ func (e *Executor) Execute(command string, env []string) error {
 		return err
 	}
 
-	go e.displayer.display(command)
+	displayFinished := make(chan bool)
```

Comment:
since `scan` is blocking, I think we don't need goroutines here. We can do something like:
```
	if err := e.displayer.startCommand(cmd); err != nil {
		return err
	}
	for e.stdoutScanner.Scan() {
		line := e.stdoutScanner.Text()
		oktetoLog.Println(line)
	}
	return cmd.Wait()
```

Reply:
That's true but we need to wait in jsonDisplayer for both(stdout and stderr) to finish

Reply:
Maybe we can use `select` there. If not, I would use goroutines only in `jsonDisplayer`